src/test/java/com/google/cloud/tools/appengine/v2/internal/cloudsdk/CloudSdkProcessFactoryFactoryTest.java

```diff
@@ -0,0 +1,130 @@
+/*
+ * Copyright 2016 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.cloud.tools.appengine.v2.internal.cloudsdk;
+
+import static org.mockito.Mockito.when;
+
+import com.google.common.collect.Maps;
+
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.TemporaryFolder;
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
+
+import java.io.IOException;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Map;
+
+public class CloudSdkProcessFactoryFactoryTest {
+
+  @Rule
+  public TemporaryFolder testFolder = new TemporaryFolder();
+
+  private Path fakeGcloud;
+  private Path fakeCredentialFile;
+
+  @Mock
+  private CloudSdkV2 sdk;
+
+  @Before
+  public void configureMocks() throws IOException {
+    MockitoAnnotations.initMocks(this);
```

Comment:
You can use `@RunWith(MockitoJUnitRunner.class)` annotation on the class to avoid explicitly calling `initMocks`